### DISCUSSION TITLE
Make pagination template vendor publishable

### DISCUSSION THREAD

**Discussion by snapey** [livewire/livewire#1934] [3. Feature ideas]
The color changes in the pagination element are too subtle, so wanting to customise this, I can't see a way to publish the template located at livewire/src/views/tailwind.blade.php ?

I did what I needed to do by copying the source from the above into a new pagination blade file and then referenced it in the component, but it would be nice to have it publishable.

**Comment by joshhanley** [comment]
I couldn't see anything to do that. I have submitted a PR #1942 to add this in.

Hope this helps!

**Comment by calebporzio** [comment]
Merged! Thanks @joshhanley !
